src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -410,7 +410,20 @@ public function load(array $configs, ContainerBuilder $container): void
         $this->registerPropertyAccessConfiguration($config['property_access'], $container, $loader);
         $this->registerSecretsConfiguration($config['secrets'], $container, $loader, $config['secret'] ?? null);
 
-        $container->getDefinition('exception_listener')->replaceArgument(3, $config['exceptions']);
+        $exceptionListener = $container->getDefinition('exception_listener');
+
+        $loggers = [];
+        foreach ($config['exceptions'] as $exception) {
+            if (!isset($exception['log_channel'])) {
+                continue;
+            }
+            $loggers[$exception['log_channel']] = new Reference(sprintf('monolog.logger.%s', $exception['log_channel']), ContainerInterface::NULL_ON_INVALID_REFERENCE);
+        }
+        
+        $exceptionListener
+            ->replaceArgument(3, $config['exceptions'])
+            ->replaceArgument(4, $loggers)
```

Comment:
```suggestion
            ->setArgument(4, $loggers)
```
Otherwise, it'll fail when using `symfony/http-kernel` < `7.3`